closure/goog/i18n/numberformat.js

```diff
@@ -462,9 +462,9 @@ goog.i18n.NumberFormat.prototype.parseNumber_ = function(text, pos) {
       normalizedText += '.';
       sawDecimal = true;
     } else if (ch == grouping.charAt(0) &&
-               ('\u00a0' != grouping.charAt(0) ||
-                pos[0] + 1 < text.length &&
-                this.getDigit_(text.charAt(pos[0] + 1)) >= 0)) {
+        ('\u00a0' != grouping.charAt(0) ||
+        pos[0] + 1 < text.length &&
+        this.getDigit_(text.charAt(pos[0] + 1)) >= 0)) {
```

Comment:
I reverted it back and it's not complaining either. :\ I'll submit it on the next commit.